### ISSUE TITLE
Fails to freeze with cx_freeze

### ISSUE THREAD

**Issue by Berserker66** [vxgmichel/aioconsole#59]
```
Traceback (most recent call last):
  File "__startup__.py", line 40, in run
  File "Console.py", line 37, in run
  File "MultiClient.py", line 16, in <module>
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 618, in _load_backward_compatible
  File "<frozen zipimport>", line 259, in load_module
  File "aioconsole\__init__.py", line 7, in <module>
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 618, in _load_backward_compatible
  File "<frozen zipimport>", line 259, in load_module
  File "aioconsole\code.py", line 36, in <module>
  File "aioconsole\code.py", line 67, in AsynchronousConsole
NameError: name 'help' is not defined
```

Line 67 in aioconsole is:

```
    @functools.wraps(help)
    def help(self, obj):
        self.print(pydoc.render_doc(obj))
```

I'm guessing the help command is not available when frozen. Would it be possible to make that command optional or some other solution?

Don't need the command myself but it makes automatic builds of the software fail.

**Comment by vxgmichel** [comment]
Hi @Berserker66, and thanks for the report!

This issue should get fixed with PR #61 and available in the next release.

However, I realized that `cx_freeze` has some problems starting new python processes and the apython script requires subprocesses in order to run with readline support. That means that `aioconsole.run_apython` most probably won't work unless the `--no-readline` option is provided.

I hope it's not an issue for you as I don't have the time required to fix this particular problem at the moment. In any case, suggestions and PR are more than welcome :)

Thanks again!

**Comment by vxgmichel** [comment]
The fix will become available with v0.1.16, thanks for the report!

**Comment by vxgmichel** [comment]
[v0.1.16](https://github.com/vxgmichel/aioconsole/releases/tag/v0.1.16) is out!
